util/caller/resolver.go

```diff
@@ -45,8 +45,20 @@ type CallResolver struct {
 }
 
 // defaultPattern strips src/github.com/organization/project/module/submodule/file.go
-// to module/submodule/file.go.
-var defaultPattern = "src/" + strings.Repeat(strings.Join([]string{"[^", "]+", ""}, string(os.PathSeparator)), 3) + "(.*)"
+// down to module/submodule/file.go.
+var defaultPattern = func() string {
+	_, file, _, ok := runtime.Caller(0)
+	if !ok {
+		panic("unable to look up location")
+	}
+	const sep = string(os.PathSeparator)
+	path := filepath.Dir(file)
+	// Strip to $GOPATH/src/.
+	for i := 0; i < 5; i++ {
+		path = filepath.Dir(filepath.Clean(path))
+	}
+	return path + sep + strings.Repeat(strings.Join([]string{"[^", "]+", ""}, sep), 3) + "(.*)"
```

Comment:
I'm not sure, but I think `path` will already have a trailing `sep` here; the final value will not have been `filepath.Clean`'d. You could then inline `sep`.